main.rb

```diff
@@ -0,0 +1,20 @@
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
+          
+        end
+end
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
```

Comment:
__[rubocop]__
* Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Method `AAA#initialize` is defined at both /sideci/sideci_rails_sandbox/main.rb:7 and /sideci/sideci_rails_sandbox/main.rb:15.

<!-- Comment by [SideCI](https://sideci.com) -->